includes/LST.php

```diff
@@ -888,4 +888,24 @@ public static function spaceOrUnderscore( $pattern ) {
 		// returns a pettern that matches underscores as well as spaces
 		return str_replace( ' ', '[ _]', $pattern );
 	}
+
+	/**
```

Comment:
Let's add quick description here about the benefits

Reply:
have a look at an elaborative essay now :)